src/Core/src/Fonts/FontManager.Windows.cs

```diff
@@ -25,18 +25,27 @@ public class FontManager : IFontManager
 		readonly IFontRegistrar _fontRegistrar;
 		readonly IServiceProvider? _serviceProvider;
 
+		/// <summary>
+		/// Creates a new <see cref="EmbeddedFontLoader"/> instance.
+		/// </summary>
+		/// <param name="fontRegistrar">A <see cref="IFontRegistrar"/> instance to retrieve details from about registered fonts.</param>
```

Comment:
Maybe "An IFontRegistrar instance for retrieving details about the registered fonts."?